COMET.Web.Common/Utilities/CherryPick/CherryPickRunner.cs

```diff
@@ -61,50 +62,80 @@ public CherryPickRunner(ISessionService sessionService)
         /// <summary>
         /// Initializes the internal properties
         /// </summary>
-        /// <param name="needCherryPicked">A collection of <see cref="INeedCherryPickedData"/></param>
-        public void InitializeProperties(IEnumerable<INeedCherryPickedData> needCherryPicked)
+        /// <param name="needCherryPickedData">A collection of <see cref="INeedCherryPickedData"/></param>
+        public void InitializeProperties(IEnumerable<INeedCherryPickedData> needCherryPickedData)
         {
-            this.NeedCherryPicked.Clear();
-            this.NeedCherryPicked.AddRange(needCherryPicked);
+            this.needCherryPicked.Clear();
+            this.needCherryPicked.AddRange(needCherryPickedData);
         }
-
+        
+        /// <summary>
+        /// Runs the cherrypick features based on data required from <see cref="needCherryPicked" />
```

Comment:
Can you specify on the doc that it runs it for all EMs that the current user is participating please?